tests/hydrothermal_cooling.prm

```diff
@@ -114,30 +103,12 @@ subsection Boundary temperature model
 end
 
 # Initial temperature field
-# Typical continental geotherm based on equations 4-6 from:
-#   D.S. Chapman (1986), "Thermal gradients in the continental crust",
-#   Geological Society of London Special Publications, v.24, p.63-70.
-# The initial constraints are:
-#   Layer Surface Temperature - upper crust (ts1) = 273 K; 
-#                               mantle      (ts3) = 823 K;  
-#   Model Base Temperature - (tb) = 1573 K;
-#   Heat Production - upper crust (A) = 1.5e-6 W/m^3; 
-#   Thermal Conductivity - upper crust (k1) = 2.5 (W/(m K)); 
-#                          lower crust (k2) = 2.5 (W/(m K)); 
-#                          mantle      (k3) = 3.3 (W/(m K));
-# To satisfy these constraints, the following values are required:
-#   Layer Surface Heat Flow - upper crust (qs1) = 0.065357 W/m^2; 
-#                             lower crust (qs2) = 0.035357 W/m^2; 
-#                             mantle      (qs3) = 0.035357 W/m^2;
-#   Temperature - base of upper crust (ts2) = 681.5714
 subsection Initial temperature model
   set Model name = function
   subsection Function
     set Variable names = x,y
-    set Function constants = h=20.e3,ts1=273,ts2=681.5714,ts3=823., tbot=1200, \
-                                     k1=2.5,k2=2.5,k3=3.3,A=1.5e-6, \
-                             qs1=0.0653571,qs2=0.035357,qs3=0.035357,qb3=0.035357
-    set Function expression = if( (h-y)<=6.e3, ts1, tbot)
+    set Function constants = h=20.e3,ts=273, tbot=1200
```

Comment:
Instead of having two constant temperatures, maybe we can have a linear temperature increase from 273 at the top to tbot at the bottom? At 20 km depth, temperature will be around 350 degrees Celsius.